docs/development/contribution-documentation/documentation-process/README.md

```diff
@@ -57,77 +57,95 @@ If you are an external contributor you don't have write permissions on the repos
 
 ![fork openproject](fork-openproject.png)
 
-## Step 6: Clone the forked OpenProject repository in GitHub Desktop
+## Step 6: Disable Actions on GitHub.com
+
+There is are automated actions configured in the original repository, that we do not need for writing documentation. Therefore we may deactivate the github actions in the Settings of our own forked repository. On GitHub.com navigate to "Settings -> Actions" and Disable Actions and confirm with **Save**.
+
+![disable-github-actions](disable-github-actions.png)
+
+## Step 7: Change the Default branch for GitHub Desktop
+
+Open `https://github.com/[your-user]/openproject`. On the forked repository go to "Settings -> Branches" in the left side menu and Switch the default branch by pressing the symbol with the **two arrows** [3].
+
+![switch-the-default-branch-step-1](switch-the-default-branch-step-1.png)
+
+Select `release/12.1` as default branch and confirm with **Update**
+
+*NOTE:* There will be an additional window. Press the button: **I understand, update the default branch.**
+
+![switch-the-default-branch-step-2](switch-the-default-branch-step-2.png)
+
+## Step 8: Clone the forked OpenProject repository in GitHub Desktop
 
 Before you can make changes you need to create a local clone of the OpenProject repository on your local computer. Open GitHub Desktop and navigate to "File -> Clone repository".
 
 ![clone repository](clone-repository.png)
 
 
 
-In the modal window select the repository you forked in Step 5. Also select a folder on your computer for the cloned repository. Click "Clone".
+In the modal window select the repository you forked in step 5. Also select a folder on your computer for the cloned repository. Click **Clone**.
 
 ![select repository to be cloned](select-repository-to-be-cloned.png)
 
-In the next screen select "*To contribute to the parent project*".
+In the next screen select **For my own purposes** and confirm with **Continue**.
```

Comment:
As pointed out below it does make sense to stick with "contribute to the parent project".
If you change your fork's default branch, new branches created in github desktop will still be based on that rather than the upstream (opf/openproject) default branch.

So I don't see a reason the choose the other behaviour here.
Here's an example of my fork where I set the default branch to release/12.**0** for testing purposes:

![image](https://user-images.githubusercontent.com/158871/174756980-45805e67-fac8-4ebc-8d9b-498a79b111a5.png)

Reply:
Hi @machisuji 

Do I understand well that you tested the complete documentation and found out that it is wrong? Or are you guessing here?
I would rather stick to what we found out before, there was a reason, why WE changed it to "For my own purposes" and it is most likely connected to changing the release branch. It was not possible in GitHub Desktop / GitHub Web otherwise, as far as I remember. I could of course try to change the release branch manually (simulate a 12.2) and see if your method works, shall I try that? I would not change these docs in this PR, before I did not test what you recommended and see myself that it really works.

Reply:
I did test it. Hence the screenshots. Feel free to do so too.

Reply:
Hi @machisuji i think the issue in GitHub Desktop after the release branch changed (we cannot test this yet before 12.2) was that the origin/release/12.1 was not yet there in GitHub Desktop

![image](https://user-images.githubusercontent.com/94119622/174980114-765508f2-b8a3-439d-bb36-0b2a6310f172.png)

Reply:
i updated the appendix, the correct screenshots are still missing for some steps, I will add them in a separate pull request, just to finish this one finally